sonar-dotnet-shared-library/src/test/java/org/sonarsource/dotnet/shared/plugins/ProtobufDataImporterTest.java

```diff
@@ -68,26 +69,32 @@ public void prepare() throws Exception {
     tester = SensorContextTester.create(new File("src/test/resources"));
     tester.fileSystem().setWorkDir(workDir);
 
-    inputFile = new TestInputFileBuilder(tester.module().key(), "Program.cs")
+    firstFile = new TestInputFileBuilder(tester.module().key(), "Program.cs")
       .setLanguage("cs")
       .initMetadata(new String(Files.readAllBytes(csFile), StandardCharsets.UTF_8))
       .build();
-    tester.fileSystem().add(inputFile);
+    tester.fileSystem().add(firstFile);
 
-    when(fileLinesContextFactory.createFor(inputFile)).thenReturn(fileLinesContext);
+    secondFile = new TestInputFileBuilder(tester.module().key(), "Data.cs")
+      .setLanguage("cs")
+      .initMetadata(new String(Files.readAllBytes(csFile), StandardCharsets.UTF_8))
+      .build();
+    tester.fileSystem().add(firstFile);
```

Comment:
```suggestion
    tester.fileSystem().add(secondFile);
```

Reply:
`secondFile` was intended here, it was a post-copy/paste issue.